src/bin/inx-chronicle/api/stardust/analytics/routes.rs

```diff
@@ -175,8 +176,24 @@ async fn richest_addresses_ledger_analytics(
         .await?
         .ok_or(ApiError::NoResults)?;
 
+    let hrp = database
+        .get_latest_protocol_parameters()
+        .await?
+        .ok_or(ApiError::Internal(InternalApiError::CorruptState(
```

Comment:
No need to manually wrap the `InternalApiError` as there is a `From` definition.